skonfig/configuration.py

```diff
@@ -0,0 +1,56 @@
+import logging
+import os
+import sys
+
+
+_logger = logging.getLogger(__name__)
+
+CONFIGURATION_DIRECTORY = "{}/.skonfig".format(os.getenv("HOME"))
+CONFIGURATION_FILE_PATH = CONFIGURATION_DIRECTORY + "/config"
+
+
+def _set_defaults(configuration):
+    configuration["beta"] = True
```

Comment:
Can we get rid of `beta`?

What is "beta" and what not looks pretty random to me and there haven't been any changes in the past few years.

Reply:
Actually, I have no idea if this has any effect today. I have been carrying this option around for years because there was always some important feature gatekeeped by it. I'll check the code...

Reply:
According to `cdist/argparse.py`:
```
# set of beta sub-commands
BETA_COMMANDS = set(('install', 'inventory', 'scan', ))
# set of beta arguments for sub-commands
BETA_ARGS = {
    'config': set(('tag', 'all_tagged_hosts', 'use_archiving', )),
}
```

We currently use "use_archiving", so maybe we need to think here a bit. Let's leave this as is here and revisit it in different issue/PR.